app/db/setting.js

```diff
@@ -0,0 +1,23 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const setting = new Schema({
+  onEventMessage: {
+    type: String,
+    default: null
+  },
+  periodicMessage: {
+    type: String,
+    default: null
+  },
+  intervalInMilliseconds: {
```

Comment:
Why not store seconds (and call it `interval`)? Would also make BE-FE communication easier. And only ws would have to be called with `interval * 1000`.

Reply:
Awesome, thanks!